net/http/client.h

```diff
@@ -25,6 +25,7 @@ limitations under the License.
 #include <photon/common/stream.h>
 #include <photon/common/timeout.h>
 #include <photon/net/socket.h>
+#include <photon/net/security-context/tls-stream.h>
```

Comment:
no need to include the header file. just make a forward declaration.

Reply:
done